src/search/operator_counting/constraint_generator.cc

```diff
@@ -6,7 +6,7 @@ using namespace std;
 
 namespace operator_counting {
 void ConstraintGenerator::initialize_constraints(
-    const shared_ptr<AbstractTask> &, named_vector::NamedVector<lp::LPConstraint> &, double) {
+    const shared_ptr<AbstractTask> &, lp::LinearProgram &) {
```

Comment:
Is this unrelated cleanup? I'm not against it, just trying to understand.

Reply:
I see now that this is related to the `infinity` cleanup.

Reply:
Part of it is unrelated cleanup, but another part is that we previously only passed the constraints to the generator. This allowed it to create its constraints but it was limited to the operator-counting variables. The new code also allows the generator to add its own local variables. The delete-relaxation constraints make heavy use of them but our previous generators all didn't need them. We only discussed the possibility of adding local variables in the paper but it was never implemented in the main branch.